src/glossary.md

```diff
@@ -0,0 +1,39 @@
+# Glossary
+
+## Toronto Community Network
+
+The name of the Toronto-based community mesh network initiative started by Toronto Mesh with its collaborators in 2020.
+
+This book documents the current resources of this initiative. For more information, visit [the project page](https://github.com/tomeshnet/toronto-community-network/).
+
+## Toronto Mesh
+
+A community group that helps communities create better networks with open source and peer-to-peer technologies that promote digital literacy and privacy.
+
+The group that started the Toronto Community Network initiative and its steward for the foreseeable future. For more information, visit [tomesh.net](https://tomesh.net).
+
+## Point-to-point (PTP)
+
+A **PTP** link is a connection that only involves two antennas. These kinds of antennas are more powerful and can reach longer distances, but only one other antenna can connect to them. They're usually implemented using highly directional (i.e. high gain) antennas and used to connect two supernodes.
+
+## Point-to-multipoint (PTMP)
+
+**PTMP** antennas support multiple links by allowing many antennas to connect to it. They're usually sector antennas (i.e. moderate gain) installed at a high point. These kinds of links are used to connect homes on the ground, because one antenna may connect to up to hundreds of others pointing to it. These kinds of links are more limited in range than PTP setups.
+
+## Node
+
+A **node** is a device that's connected and accessible on the mesh network. This might be a router and antenna combo, for example, on top of someone's house, or an apartment building. This doesn't include devices connected to the mesh network that are [NATed](https://en.wikipedia.org/wiki/Network_address_translation), so even if your home router is connected to the mesh network, your laptop and phone usually aren't counted as nodes.
+
+## Supernode
+
+A **supernode** is an actively maintained relaying node that supports high bandwidth point-to-multipoint (PTMP) connectivity. Currently all supernodes are managed by Toronto Mesh, but other groups may also operate their own supernodes on the Toronto Community Network.
+
+## Babel
+
+The **Babel routing protocol** is what we use to create the mesh network. It's the software that allows the "meshing" to happen, by routing packets where they need to go.
```

Comment:
Babel does not actaully route any packets. The router does that. Babel dynamically installs routes into the operating system after deciding which direction is the best next hop  packets should take for each destination/subnet.

Once the routes are installed, the operating system uses them as if they where provided by any other source.

Reply:
Yes, this is a simplification, as I figured many people will not understand routing rules or route tables. If you have a suggested change I'd be happy to look at it but I'm okay with this simplification.

Reply:
My issue is that this simplification presents babel as something it is not - a router. IE CJDNS is a router. Babel has no interaction with the packets that cross the mesh. In fact the underlying can told not to list for babel.

The line also confuses BABEL (a protocol) with BABELD (a software). (HTTP is a protocol. Firefox is software)

If we are looking for simplifications it would be 
`The **Babel routing protocol** is a protocol for create a layer 3 mesh network. `

I don't know how to summarize it in a line or two. I think i need to expand it on my babeld page i been writing.

But maybe.

```suggestion
The **Babel routing protocol** is a protocol that is used to create a layer 3 mesh network. By running a babel implementations on nodes of a network, routes (the next direction a packet should take) to all the nodes will be actively calculated and implement on those nodes.
```

Reply:
I committed this one, what do you think?
> `The **Babel routing protocol** is used to create a mesh network. By running a Babel implementations on the nodes of the network, paths to all other nodes are actively calculated, so that connections can be made to any node on the mesh.`